iac-common/src/main/java/org/sonar/iac/common/yaml/TreePredicates.java

```diff
@@ -35,4 +39,10 @@ public static Predicate<YamlTree> isTrue() {
   public static Predicate<YamlTree> isEqualTo(String parameter) {
     return t -> TextUtils.isValue(t, parameter).isTrue();
   }
+
+  public static Predicate<YamlTree> isSet() {
+    return t -> TextUtils
+      .getListValueElements(t).stream()
```

Comment:
See the comment at the `getListValueElements` declaration. The predicate
`TextUtils.matchesValue(t, value -> !STRINGS_CONSIDERED_AS_EMPTY.contains(value)).isTrue()` does the same job.